src/routes/recipes.js

```diff
@@ -0,0 +1,60 @@
+import { Router } from 'express';
+import { query, validationResult } from 'express-validator';
+
+import { authenticate } from '../modules/auth';
+import models from '../modules/database';
+import loggers from '../modules/logging';
+
+const router = Router();
+const log = loggers('recipes');
+const { Recipe } = models;
+
+/**
+ * GET Recipes
+ * @query offset int
+ * @query limit int
+ */
+router.get(
+  '/',
+  authenticate(),
+  [
+    query('offset')
+      .optional()
+      .isNumeric()
+      .toInt(),
+    query('limit')
+      .optional()
+      .isNumeric()
+      .toInt()
+  ],
+  async (req, res) => {
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty()) {
+      return res.status(400).json({ errors: errors.array() });
+    }
+    const limit = req.query.limit || 20;
+
+    const offset = req.query.offset - 1 || 0;
+
+    log.info(`request for recipes ${limit}`);
```

Comment:
This could be `Request for ${limit} recipes starting from ${offset}` - but also I think we should be logging this kind of thing at the debug level rather than info - imo info is only for things that you would want to see while looking through the logs of a running prod instance, and this isn't very important for that.